internal/controller/observability/collector.go

```diff
@@ -139,11 +139,19 @@ func ReconcileCollector(context internalcontext.ForwarderContext, pollInterval,
 		log.Error(err, "collector.ReconcileInputServices")
 		return err
 	}
-
+	networkPolicyName := fmt.Sprintf("%s-%s", constants.CollectorName, resourceNames.CommonName)
```

Comment:
nit.  This can be moved inside the "if"

Reply:
This cannot be moved because this policy name is needed for the `else` part of this statement for removal of the NetworkPolicy when the spec is removed on line 151.